packages/api-client/src/main/auth/AccessTokenStore.ts

```diff
@@ -31,54 +29,19 @@ class AccessTokenStore extends EventEmitter {
 
   public accessToken: AccessTokenData | undefined;
 
-  public static TOPIC = {
-    ACCESS_TOKEN_REFRESH: 'AccessTokenStore.TOPIC.ACCESS_TOKEN_REFRESH',
-  };
-
-  constructor(private engine: CRUDEngine) {
-    super();
-  }
-
   public async delete(): Promise<void> {
-    this.logger.info(
-      `Deleting access token in store "${
-        this.engine.storeName
-      }" on table "${AUTH_TABLE_NAME}" with key "${AUTH_ACCESS_TOKEN_KEY}"`
-    );
-    return this.engine.delete(AUTH_TABLE_NAME, AUTH_ACCESS_TOKEN_KEY).then(() => (this.accessToken = undefined));
+    this.logger.info('Deleting access token');
+    this.accessToken = undefined;
+    return Promise.resolve();
   }
 
   public async updateToken(accessToken: AccessTokenData): Promise<AccessTokenData> {
     if (this.accessToken !== accessToken) {
-      this.logger.info(
-        `Updating access token in store "${
-          this.engine.storeName
-        }" on table "${AUTH_TABLE_NAME}" with key "${AUTH_ACCESS_TOKEN_KEY}"`
-      );
-      return this.engine
-        .updateOrCreate(AUTH_TABLE_NAME, AUTH_ACCESS_TOKEN_KEY, accessToken)
-        .then(() => (this.accessToken = accessToken));
+      this.logger.info('Updating access token');
+      this.accessToken = accessToken;
     }
     return Promise.resolve(this.accessToken);
   }
-
-  public async init(): Promise<AccessTokenData | undefined> {
-    this.logger.info(
-      `Initialising access token from store "${
-        this.engine.storeName
-      }" on table "${AUTH_TABLE_NAME}" with key "${AUTH_ACCESS_TOKEN_KEY}"`
-    );
-    return this.engine
-      .read<AccessTokenData>(AUTH_TABLE_NAME, AUTH_ACCESS_TOKEN_KEY)
-      .catch((error: Error) => {
-        if (error.name === RecordNotFoundError.name) {
-          return undefined;
-        }
-
-        throw error;
-      })
-      .then((accessToken: AccessTokenData | undefined) => (this.accessToken = accessToken));
-  }
```

Comment:
This is a major change, removing this will break a lot of other packages.

Reply:
It's not because the `AccessTokenStore` is private in the `Client`. As long as the api-client itself is not using this function it's not a breaking change (and we never use it in the api-client).